### ISSUE TITLE
Product Research

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/door#13]
If you want to help, please add comments to this issue with commercially available products in the Door/Lock market. e.g:

Price From: £319.49 + VAT ...
https://www.locksonline.co.uk/Wireless-Digital-Touchscreen-Glass-Door-Lock.html
![image](https://user-images.githubusercontent.com/194400/37716979-a91b65b6-2d17-11e8-8ad5-c304beab7ff7.png)

**Comment by nelsonic** [comment]
Sliding Doors: List: https://www.ironmongerydirect.co.uk/browse/sliding-door-hardware/interior-timber-sliding-door-systems

https://www.ironmongerydirect.co.uk/product/kl-g-straight-sliding-door-track-70-100kg-3000mm-262348
![image](https://user-images.githubusercontent.com/194400/46276434-f949f380-c557-11e8-80d5-7a3898f18160.png)

**Comment by nelsonic** [comment]
Sprung Hinge for self-closing (bedroom) doors:
https://www.ironmongerydirect.co.uk/product/adjustable-sprung-hinge-100x75mm-satin-chrome-pack-3-812162
![image](https://user-images.githubusercontent.com/194400/46347066-25da3a00-c642-11e8-8a54-86ab6b6e40a2.png)

More: https://www.ironmongerydirect.co.uk/search?SearchText=self-closing

**Comment by iteles** [comment]
Andy and Madalena got theirs from http://www.jnf.pt/pt/produtos/sistemas-de-controlo-de-acessos and they love them! In particular the card making apparatus is great and the application to manage the bookings is also very good.

![image](https://user-images.githubusercontent.com/4185328/46350727-6be8cb00-c64d-11e8-8de4-d1e549aa094f.png)



I think this one in particular:
![image](https://user-images.githubusercontent.com/4185328/46350710-60959f80-c64d-11e8-8a94-2b139d5c5dcb.png)


I don't have the price yet but I remember Andrew mentioning just over €100 previously, maybe €120 each.

**Comment by nelsonic** [comment]
Yep, these are *exactly* the kind that are vulnerable to being _cracked_ with off-the-shelf readers. #8 
In fact for the price of "dinner out" you could buy this reader/copier: https://www.aliexpress.com/item/English-10-frequency-RFID-Copier-ID-IC-Reader-Writer-copy-M1-13-56MHZ-encrypted-Duplicator-Programmer/32740178133.html 
![image](https://user-images.githubusercontent.com/194400/46355799-6b563180-c659-11e8-9361-86c7f55c0b9a.png)

Walk past someone who has the master key and clone their key, gaining access to the entire building.
and since their system is not integrated/connected the key cannot be _revoked_. 😞 
It's _massively_ reliant on "trust". all it would take is one "Felipe Cunha" to run _riot_ in the place.

**Comment by nelsonic** [comment]
Sprung Hinges for auto-closing doors:
https://www.ironmongerydirect.co.uk/browse/hinges/sprung-hinges

**Comment by nelsonic** [comment]
For keeping track of our books: https://github.com/dwyl/hq/issues/482 
Thinking of using these RFID stickers, e.g: 100PCS/lot Long Range RFID UHF Tag Sticker https://www.aliexpress.com/item/4000092722305.html
![image](https://user-images.githubusercontent.com/194400/86797440-91e3a080-c067-11ea-88d4-791ea703f189.png)

These are obviously cheaper (5cents each when bought in bulk):
https://www.aliexpress.com/item/4000092685607.html
![image](https://user-images.githubusercontent.com/194400/86797947-2c43e400-c068-11ea-8b97-80d2091da112.png)

https://www.aliexpress.com/item/32267332202.html
![image](https://user-images.githubusercontent.com/194400/86798197-6f05bc00-c068-11ea-9b7e-14947d9245b9.png)
